src/garage/sampler/ray_sampler.py

```diff
@@ -135,7 +148,11 @@ def _process_trajectory(self, result):
         Converts that trajectory to garage friendly format.
 
         Args:
-            - result: ray object id of ready to be collected trajectory.
+            result (int): ray object id of ready to be collected trajectory.
```

Comment:
though it's appreciated, you should be able to skip docstrings on _private methods

Reply:
Pylint is forcing this as well